tests/testthat/test-roundtrip-obsmvarm.R

```diff
@@ -129,4 +130,40 @@ for (name in test_names) {
       py_get_item(adata_py$varm, name)
     )
   })
+
+  skip_if_no_h5diff()
+  # Get all R datatypes that are equivalent to the python datatype (name)
+  res <- Filter(function(x) x[[1]] == name, all_equivalences)
+  r_datatypes <- sapply(res, function(x) x[[2]])
+
+# nolint start
+  for( r_name in r_datatypes){
+    test_that(paste0("Comparing a python generated .h5ad with obsm and varm '", name,
```

Comment:
Why do you use nolint start here?

Please reformat :bow: You can use `styler::style_pkg()` if you like

```suggestion
  for (r_name in r_datatypes) {
    test_that(paste0("Comparing a python generated .h5ad with obsm and varm '", name,